google-cloud-clients/google-cloud-core/pom.xml

```diff
@@ -38,6 +38,11 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.http-client</groupId>
+      <artifactId>google-http-client-apache</artifactId>
```

Comment:
Can this be moved to specific projects?  I'd rather reduce dependencies, if possible.